packages/tools/src/tools/annotation/planarFreehandROITool/renderMethods.ts

```diff
@@ -51,7 +52,25 @@ function renderContour(
   annotation: PlanarFreehandROIAnnotation
 ): void {
   if (annotation.data.isOpenContour) {
-    this.renderOpenContour(enabledElement, svgDrawingHelper, annotation);
+    if (annotation.data.isOpenCardiacAnnotation) {
+      if (!annotation.data.openCardiacAnnotationVectorToPeak) {
```

Comment:
Not important, but this can be better re-written to be more readable with gaurd clauses

Reply:
Simplified a little and added comments.